src/token_classifier.rs

```diff
@@ -1,17 +1,22 @@
-use std::collections::HashSet;
-
 use deunicode::deunicode_char;
+use fst::Set;
 
 use crate::{Token, TokenKind};
 use crate::token::SeparatorKind;
 
 #[derive(Debug,  Clone)]
-pub struct TokenClassifier<'a> {
-    stop_words: &'a HashSet<String>,
+pub struct TokenClassifier<'a, A>
+where
+    A: AsRef<[u8]>
```

Comment:
Same here and everywhere else, no need for this bound. :)

Reply:
I will look at that! Thanks mops 😊